WordPress/Classes/ViewRelated/People/PeopleViewController.swift

```diff
@@ -497,8 +503,9 @@ private extension PeopleViewController {
         filterBar.items = filtersAvailableForBlog(blog)
         filterBar.addTarget(self, action: #selector(selectedFilterDidChange(_:)), for: .valueChanged)
 
-        // By default, let's display the Blog's Users
-        filter = .users
+        let indexToSet = Filter.allCases.firstIndex(where: { $0 == defaultFilter }) ?? 0
```

Comment:
nitpick:
```suggestion
        let indexToSet = Filter.allCases.firstIndex(of: defaultFilter) ?? 0
```

Also, I think a force unwrap is OK since we know that `defaultFilter` is present in `allCases`, but that's questionable.